specification/sections/03-evaluation-context.md

```diff
@@ -30,6 +30,8 @@ The targeting key uniquely identifies the subject (end-user, or client service)
 
 > The evaluation context **MUST** support the inclusion of custom fields, having keys of type `string`, and values of type `boolean | string | number | datetime | structure`.
 
+Each key should be unique.
```

Comment:
Please move this to a new requirement called "Requirement 3.1.3" and make sure the requirement is under a markdown quote block. You'll need to run `make parse` to update the requirements JSON file.

Please also define the behavior of a collision.

Reply:
I just noticed that there's a PR to introduce 3.1.3. In that case, perhaps use 3.1.4. Having a dedicated section will make it easier for developers to confirm that they've explicitly tested this behavior.